domains/emissions/Emissions.API/chart-overrides.yaml

```diff
@@ -1,4 +1,4 @@
-version: 0.3.7
+version: 0.3.8
```

Comment:
I would argue these changes merit a major version change instead of a patch.

Reply:
bumped in 18708ff